.gitignore

```diff
@@ -7,4 +7,4 @@ node_modules
 .idea
 .idea/workspace.xml
 node_modules/
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
```

Comment:
this was part of a previous PR.
any particular reason this is still here ?

Reply:
I added the new line mentioned in the previous PR. Hence, the change was showed.